source/index.html

```diff
@@ -4,17 +4,186 @@
+              <!--! Q нужен ли класс-модификатор для картинки?-->
+              <h4 class="visually-hidden">сплошная польза</h4>
+              <!--! Q нужен ли здесь класс advantages__title?-->
+              <p class="advantages__description">Функциональное питание
+                <br>содержит только полезные
+                <br>питательные вещества.</p>
+            </li>
+            <li class="advantages__item advantages__item--easy">
+              <h4 class="visually-hidden">легко готовится</h4>
+              <p class="advantages__description">Выпускается в виде порошка,
+                <br>который нужно лишь залить
+                <br>кипятком и готово.</p>
+            </li>
+            <li class="advantages__item advantages__item--diet">
+              <h4 class="visually-hidden">диета##</h4>
+              <!--! Q затрудняюсь придумать название заголовку-->
+              <p class="advantages__description">Замените один-два приема
+                <br>обычной еды на наше
+                <br>функциональное питание.</p>
+            </li>
+            <li class="advantages__item advantages__item--result">
+              <h4 class="visually-hidden">наглядный результат</h4>
+              <p class="advantages__description">Уже через месяц
+                <br>наслаждайтесь изменениями <br>к лучшему вашего питомца!</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="stats"> <!--! Q блок статистики?-->
+        <div class="stats__wrapper slider">
+          <!--! не понимаю логику слайдера((-->
+          <header class="stats__header">
+            <h2 class="stats__title">Живой пример</h2>
+            <p class="stats__text">Борис сбросил 5 кг за 2 месяца, просто
+              <br>заменив свой обычный корм на Cat Energy <br>Slim. Отличный
+              результат без изнуряющих <br>тренировок! При этом он не менял
+              своих <br>привычек и по-прежнему спит по 16 часов <br>в день.
+            </p>
+          </header>
+          <div class="stats__wrapper">
+            <!--! враппер нужен для стилизации Веса и Времени-->
+            <div class="stats__weight">
+              <b>5 кг</b>
+              <br>
+              <small>снижение<br> веса</small>
+            </div>
+            <div class="stats__time">
+              <b>60 дней</b>
+              <br>
+              <small>затрачено<br> времени</small>
+            </div>
+          </div>
+          <div class="stats__price">
+            <span>затраты на питание: 15 000 руб.</span>
+          </div>
+          <img src="img/compare-slider.jpg"
+            alt="Борис до и после курса питания.">
+          <!-- ЗДЕсь должен быть сверстан слайдер-->
+        </div>
+      </section>
+      <section class="contacts">
+        <h2 class="visually-hidden">Свяжитесь с нами</h2>
+        <b class="contatts__title">приглашаем <br>к сотрудничеству
+          <br>дилеров!</b>
```

Comment:
```suggestion
        <p class="contacts__title">приглашаем к сотрудничеству
          дилеров!</p>
```
И тут бы поправить логику.
Ты вроде считаешь этот текст заголовком, даешь ему класс title, но при этом используешь тег `p`. Либо туда либо сюда.